edx_proctoring/templates/proctored_exam/instructions.html

```diff
@@ -4,7 +4,7 @@
   <div class="">
     <h3>
     {% blocktrans %}
-      Follow these steps to set up and start your proctored exam.
+      Set up and start your proctored exam.
```

Comment:
@stroilova please note that this change will apply to both this page and the Instructions page used for Proctortrack exams.

All of the other changes to the copy on this page apply only to PSI exams, and not to Proctortrack ones.